modules/ROOT/pages/virtual-private-cloud.adoc

```diff
@@ -5,10 +5,10 @@ endif::[]
 
 The Virtual Private Cloud (VPC) offering allows you to create a virtual, private, and isolated network segment in the cloud to host your CloudHub workers.
 
-By connecting to your CloudHub VPC you can extend your corporate intranet whether on-prem data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
+By connecting to your CloudHub VPC you can extend your corporate intranet whether on-premises data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
```

Comment:
This sentence is a bit long/hard to parse. Maybe rearrange the two sentences so something like "Connecting..your corporate intranet allows CloudHub workers to access.... You can connect whether you use on-premise data centers through a tunnel..." (separate the two key ideas: you can do this, and you can do this in three different ways"